conftest.py

```diff
@@ -221,12 +273,49 @@ def _benchmark_time():
 
 @pytest.fixture(scope="function")
 def auto_benchmark_time(benchmark_time):
+    """Benchmark the wall clock time of an individual test.
+
+    This fixture automatically records the wall clock time it takes to
+    execute an individual test if run as part of a benchmark. Depending on the test's structure, this may
+    include setup or teardown code that should not be measured. For more
+    control, use the ``benchmark_time`` context manager fixture.
+
+
+    Example
+    -------
+    .. code-block:: python
+
+        def test_something(auto_benchmark_time):
+            do_something()
+
+    See Also
+    --------
+    benchmark_time
+    """
     with benchmark_time:
         yield
 
 
 @pytest.fixture(scope="function")
 def benchmark_memory(test_run_benchmark):
+    """Benchmark the memory usage of executing some code.
+
+    Yields
+    ------
+    Context manager factory function which takes a ``distributed.Client``
+    as input. The context manager records peak and average memory usage of
+    executing the ``with`` statement if run as part of a benchmark,
+    or does nothing otherwise.
+
+    Example
+    -------
+    .. code-block:: python
+
+        def test_something(benchmark_memory):
+            with benchmark_memory(client):
```

Comment:
Here and below: you restore the `client` creation as in the README to make the example more explicit? Otherwise, no objection from me.